polaris.shopify.com/src/components/SearchField/SearchField.tsx

```diff
@@ -7,14 +9,18 @@ interface Props extends Omit<HTMLProps<HTMLInputElement>, "onChange"> {
 
 function SearchField({ onChange, ...props }: Props) {
   return (
-    <input
-      type="search"
-      className={styles.SearchField}
-      onChange={(evt) => {
-        onChange && onChange(evt.target.value);
-      }}
-      {...props}
-    />
+    <div className={styles.SearchField}>
+      <div className={styles.Icon}>
+        <Image src={filterMajor} width={20} height={20} alt="" icon />
```

Comment:
We can probably replace this with `/icons/whateversearchiscalled.svg` but this is super.